pyEpiabm/pyEpiabm/routine/place_forces.py

```diff
@@ -0,0 +1,65 @@
+#
+# Calculate place force of infection based on Covidsim code
+#
+
+from pyEpiabm.core import Person, Place
+
+
+class PlaceForces:
+    """Class to calculate the infectiousness and susceptibility
+    parameters for the force of infection parameter, within places.
+    """
+
+    @staticmethod
+    def place_susc(place: Place, infector: Person, infectee: Person,
+                   timestep: int):
+        """Calculate the susceptibility of a place.
+
+        :param infector: Infector
+        :type infector: Person
+        :param infectee: Infectee
+        :type infectee: Person
+        :param place: Place
+        :type place: Place
+        :param timestep: Current simulation timestep
+        :type timestep: int
+        :return: Susceptibility parameter of place
+        :rtype: float
+        """
+        return 0.2
+
+    @staticmethod
+    def place_inf(place: Place, timestep: int):
+        """Calculate the infectiousness of a place.
+        Not dependent on the people in it.
+
+        :param place: Place
+        :type place: Place
+        :param timestep: Current simulation timestep
+        :type timestep: int
+        :return: Infectiousness parameter of place
+        :rtype: float
+        """
+        return 0.5
+
+    @staticmethod
+    def place_inf_force(place: Place, infector: Person, infectee: Person,
```

Comment:
Thought for later: we probably want to have a baseline susc, inf, foi and then possibility of changing it within a simulation (ie if travel restrictions come in foi =0 etc). Could we have a  place_base_foi and separate method place_adapt_foi (and maybe another separate place_reset_foi) which would take the baseline and change it

Reply:
I agree, but think this might be too much future engineering; these won't be hard to add later so better to do it then when we know what we want!